CHANGELOG.md

```diff
@@ -1,6 +1,19 @@
 # koa-oas3
 
+## 0.16.0
```

Comment:
We don't need to manually update this CHANGELOG. It will be automatically populated during the release.

Reply:
Hey @crizo23 would you be able to remove the changes of this file from the commit?

Reply:
Fixed, thanks @supertong